Tests/TSCBasicTests/ProcessTests.swift

```diff
@@ -42,20 +42,57 @@ class ProcessTests: XCTestCase {
     }
 
     func testPopen() throws {
+        #if os(Windows)
+        let echo = "echo.exe"
+        #else
+        let echo = "echo"
+        #endif
         // Test basic echo.
-        XCTAssertEqual(try Process.popen(arguments: ["echo", "hello"]).utf8Output(), "hello\n")
+        XCTAssertEqual(try Process.popen(arguments: [echo, "hello"]).utf8Output(), "hello\n")
 
         // Test buffer larger than that allocated.
         try withTemporaryFile { file in
             let count = 10_000
             let stream = BufferedOutputByteStream()
             stream <<< Format.asRepeating(string: "a", count: count)
             try localFileSystem.writeFileContents(file.path, bytes: stream.bytes)
-            let outputCount = try Process.popen(args: "cat", file.path.pathString).utf8Output().count
+            #if os(Windows)
+            let cat = "cat.exe"
```

Comment:
Similar to echo, `cat.exe` is not a thing.  The equivalent to `cat` is `type` which is a shell builtin.